homeassistant/components/fan/__init__.py

```diff
@@ -235,6 +235,13 @@ class FanEntity(ToggleEntity):
     """Base class for fan entities."""
 
     entity_description: FanEntityDescription
+    _attr_current_direction: str | None = None
+    _attr_oscillating: bool | None = None
+    _attr_percentage: int | None = None
+    _attr_preset_mode: str | None = None
+    _attr_preset_modes: list[str] | None = None
+    _attr_speed_count: int | None = None
```

Comment:
`None` is not valid `speed_count`?

Reply:
would say no, since it always returns an `int`:
https://github.com/home-assistant/core/blob/3175bca37d1af22327ca506ea214157af57acbc5/homeassistant/components/fan/__init__.py#L482-L488

Reply:
ah ... now i get your point 🙈